util/buildRelease/testRelease

```diff
@@ -7,48 +7,12 @@ $cwd = abs_path(dirname(__FILE__));
 $chplhomedir = abs_path("$cwd/../..");
 
 
-# Mailing lists.
-$failuremail = "chapel+tests\@discoursemail.com";
-$allmail = "chapel-test-results-all\@lists.sourceforge.net";
-$replymail = "";
-
-$printusage = 1;
-$debug = 1;
```

Comment:
Looks like this PR also removes support for `-debug` from this script. Can you highlight that in the PR message?

Reply:
(it removes `-debug` and `-cron`)

Reply:
Added. (I removed those arguments because as far as I could tell they only control emailing behavior.)